### PR TITLE
Search revamp

### PR DESCRIPTION
**Pull request by akshaaatt**
- Search UX & UI Upgrade
- Theming Updated
- Bug fixes
- Incorporated Design suggestions from the Community at MetaBrainz

### CODE REVIEW

**Comment by akshaaatt** [comment]
@amCap1712 I think we are ready to check this out. The design suggested by people at community.meb has been incorporated in the artist bio fragment.

File: app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -7,9 +7,11 @@
         <item>@string/search_type_release</item>
         <item>@string/search_type_label</item>
         <item>@string/search_type_recording</item>
+        <item>@string/search_type_event</item>
         <item>@string/search_type_release_group</item>
         <item>@string/search_type_instrument</item>
-        <item>@string/search_type_event</item>
+        <item>Area</item>
```
Comment (amCap1712) [inline app/src/main/res/values/arrays.xml:13]: We do not support searching this types yet, so why these here?

**Review by amCap1712 — CHANGES_REQUESTED** [review]
LGTM, except the one comment. Works splendidly thanks!

File: app/src/main/res/values/arrays.xml (hunk 1 above)
Comment (akshaaatt) [reply app/src/main/res/values/arrays.xml:13]: Sure, while I was working for the UX part, I added it to see how things would look like. Will remove it for now!

**Review by amCap1712 — APPROVED** [review]

**Pull request merged**